cmd/kcp-front-proxy/main.go

```diff
@@ -73,6 +73,7 @@ func main() {
 
 func NewProxyCommand(ctx context.Context) *cobra.Command {
 	options := frontproxyoptions.NewOptions()
+	logger := klog.FromContext(ctx)
```

Comment:
Can we add a `WithValues("component", "front-proxy")` here to help distinguish this in a communal log bucket?